docs/docs/tutorials/fashion-mnist-yarn-spark_submit.md

```diff
@@ -0,0 +1,267 @@
+
+To submit the BigDL program to Yarn with `spark_submit`, we need to specify the `cluster_mode` to `spark-submit`. In this case, we should set the Spark configurations through command-line options or the properties file.
+
+```python
+from bigdl.orca import init_orca_context, stop_orca_context
+
+init_orca_context(cluster_mode="spark-submit")
+```
+
+## 5. Create the PyTorch Estimator
+
+After initializing an OrcaContext, we could simply create a PyTorch estimator, and the Orca Estimator will replicate the model on each node in the cluster, feed the data partition on each node to the local model replica, and synchronize model parameters using various backend technologies.
+
+```python
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.metrics import Accuracy
+
+orca_estimator = Estimator.from_torch(model=model_creator,
+                                      optimizer=optimizer_creator,
+                                      loss=criterion,
+                                      metrics=[Accuracy()],# Orca validation methods for evaluate.
+                                      model_dir=os.getcwd(),# The path to save model
+                                      backend="spark")
+```
+
+The PyTorch Estimator supports backends including spark, ray, bigdl ,and horovod.
+
+`Note`: The backend of Estimator must not be `bigdl` when we are running with `spark-submit`. It's better to use the `bigdl` backend when we initialize  OrcaContext with `yarn-client` mode and run the program with the built-in function.
+
+## 6. Training and Testing the Model
+
+It's time to call `Estimator.fit` to train the network on the training dataset, we can simply feed the input to the model and set times to loop over input. 
+
+```python
+stats = orca_estimator.fit(train_data_creator, epochs=epochs, batch_size=batch_size)
+print("Train stats: {}".format(stats))
+```
+To evaluate the network on the testing dataset, we can call `Estimator.evaluate`, which will return a dictionary of metrics for the given data, including validation accuracy and loss.
+
+```python
+val_stats = orca_estimator.evaluate(validation_data_creator, batch_size=batch_size)
+print("Validation stats: {}".format(val_stats))
+```
+
+# Run BigDL Program on Yarn with Spark-Submit
+
+The BigDL supports submitting an application to the Yarn cluster, we could refer to the following steps in order:
+
+1. Pack the Conda environment to an archive.
+
+```bash
+conda pack -f -o environment.tar.gz
+```
+
+This archive file captures the Conda environment for Python and stores both Python interpreter and all its relevant dependencies, we can ship it together with scripts by using the `--archives` option, i.e. `--archives environment.tar.gz#environment`. It will be automatically unpacked on executors.
```

Comment:
what do you mean by `ship it together with scripts by using the --archives option`?

Reply:
Like sending all things packed in the conda archive file to YARN through `--archives` in the `spark-submit` script.